tests/jsondb/test_jsonfiledb.py

```diff
@@ -9,7 +9,7 @@ def test_jsonfiledb__get_uri_for_file(tmp_path):
         db: AerovalJsonFileDB
         assert (
             db._get_uri_for_file(str(tmp_path / "project/experiments.json"))
-            == "/v0/experiments/project?version=0.0.1"
+            == '/v0/experiments/"project"?version=0.0.1'
```

Comment:
why is it `"project"` and not, if you indent to say that project is a placeholder `{project}`